src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowRequestTree.java

```diff
@@ -353,6 +388,91 @@ public void updateElement(final int index, final Pane rowPane)
         });
     }
 
+    /**
+     * Checks if the request is fulfillable
+     *
+     * @param tRequest the request to check if it's fulfillable
+     */
+    public Boolean fulfillable(final IRequest tRequest)
+    {
+        if (!(tRequest.getRequest() instanceof IDeliverable))
```

Comment:
Shouldn't we call this method now in the original research tree too? To avoid duplicate code